simplePlayground.html

```diff
@@ -74,28 +74,49 @@ <h2>Tesla Ib874</h2>
         </div>
     </section>
 
+    <section class="speed">
+        <h1>Text</h1>
+        <p>This section demonstrates the <b>Inline text semantics</b></p>
+
+        <img src="images\gncap.jpg" alt="This car have 5 stars rating on GNCAP">
+
+        <div class="product">
+            <img src="images/sample-image.jpg" alt="Product Image" title="Product model Ib874" />
+            <h2>Tesla IH125</h2>
+            <p>
+                This is a smaller version of the Ib874 but with almost the same capabilities.
+            </p>
+            <p>
+                Before: <span class="before-price">$500</span>
+            </p>
+            <p>
+                Now: <span class="current-price">$400</span>
+            </p>
+            <button>Buy now</button>
+        </div>
+    </section>
+
     <!-- This section is for List examples -->
     <section>
         <h1>List</h1>
         <h3>Calls procedure checklist</h3>
         <ol>
-            <li><br>Answer</b> call</li>
-            <li><b>Ask</b> for insurance</li>
-            <lr><b>Ask</b> for injuries</li>
-            <li><b>Find</b> a doctor nearby</li>
-            <li>
+            <li><b>Answer</b> call</li>
+            <li><b>Ask</b> for <em>insurance</em></li>
+            <li><b>Ask</b> for injuries</li>
+            <li><b>Find</b> a doctor <em>nearby</em></li>
+        </ol>
                 Complaint procedure
                 <ul>
                     <li>Fill complaint form</li>
-                    <li>Call boss</li>
                     <li>Hangout</li>
+                    <li>Call boss</li>
                 </ul>
-            </li>
         </ol>
         <h3>TODO List</h3>
         <ul>
-            <li>Call Mom</li>
-            <lt>Buy tickets</lt>
+            <li><del>Call Mom</del></li>
+            <li><del>Buy tickets</del> </li>
```

Comment:
Personalmente usaría una clase para el elemento de la lista, por ejemplo:
```suggestion
            <li class="done">Buy tickets</li>
```
En donde ese estilo por ahora solo tendría `text-decoration: line-through;` pero a futuro puede incluir más estilos a nivel de bloque que van más allá de lo que la etiqueta **del** tiene.
En cuanto a la semántica, creo que en este caso tiene más sentido decir "**done**" que decir "**deleted**"